core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -16,161 +16,208 @@
  * @fileoverview Unit tests for the about page.
  */
 
+import { ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import { EventEmitter, NO_ERRORS_SCHEMA, Pipe, PipeTransform }
+  from '@angular/core';
+
+import { AboutPageComponent } from './about-page.component';
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { TranslateService } from 'services/translate.service';
+import { UtilsService } from 'services/utils.service';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
 import { WindowRef } from 'services/contextual/window-ref.service';
 
-require('pages/about-page/about-page.component.ts');
+@Pipe({name: 'translate'})
+class MockTranslatePipe implements PipeTransform {
+  transform(value: string, ...args):string {
+    return value;
+  }
+}
+
+class MockTranslateService {
+  code = 'es';
```

Comment:
This is just a dummy value. If you see the last test in the I am emitting a code change event and if that is successfully captured then the value should be `en` instead of `es`.